tests/console/w3m_https.pm

```diff
@@ -22,7 +22,7 @@ sub run {
     select_console "root-console";
 
     assert_script_run('rpm -q w3m');
-    script_run("zypper --no-refresh se -it pattern fips");
+    script_run("zypper --no-refresh --non-interactive se -it pattern fips");
```

Comment:
why not use `zypper_call` which use `--non-interactive` by default

Reply:
Sorry, my mistake, use zypper_call instead.
Verification run updated: http://10.67.17.9/tests/22#step/w3m_https/4